tests-beta/ios/MASVS-RESILIENCE/MASTG-TEST-0x89.md

```diff
@@ -0,0 +1,32 @@
+---
+platform: ios
+title: Runtime Use of Jailbreak Detection Techniques
+id: MASTG-TEST-0x89
+type: [dynamic]
+weakness: MASWE-0097
+false_negative_prone: true
+---
+
+## Overview
+
+The test verifies that a mobile application can identify if the iOS device it is running on is jailbroken. It does so by dynamically analyzing the app binary for [common jailbreak detection checks](../../../Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md#common-jailbreak-detection-checks) and trying to bypass them. For example, it may detect a check for the presence of certain files or directories that are indicative of a jailbroken device and use dynamic instrumentation to bypass them by modifying the list of files or directories that are checked.
```

Comment:
```suggestion
The test verifies that a mobile application can identify if the iOS device it is running on is jailbroken. It does so by dynamically analyzing the app binary for [common jailbreak detection checks](../../../Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md#common-jailbreak-detection-checks) and trying to bypass them. For example, it may detect a check for the presence of certain files or directories that are indicative of a jailbroken device.
```
Removed the last sentence as the bypass angle is irrelevant. We're using the tool to collect artefacts, not to bypass a JB check